analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/HardcodedIpAddressBase.cs

```diff
@@ -100,6 +115,18 @@ private void CheckForHardcodedIpAddressesInStringInterpolation(SyntaxNodeAnalysi
             && !IPAddress.IsLoopback(ipAddress)
             && !ipAddress.GetAddressBytes().All(x => x == 0); // Nonroutable 0.0.0.0 or 0::0
 
+        private bool IsInDocumentationBlock(IPAddress address) =>
+            address switch
+            {
+                { AddressFamily: AddressFamily.InterNetwork } when
+                    address.GetAddressBytes() is { } parts
```

Comment:
This should help with the coverage issue, also below. GetAddressBytes should not return null.
```suggestion
                    address.GetAddressBytes() is var parts
```